index.js

```diff
@@ -20,6 +20,8 @@ module.exports = app => {
 
     const branch = ref.replace(/^refs\/heads\//, '')
 
+    const targetBranch = config['target-branch']

```

Comment:
with schema in #193 this empty string would be the default

```suggestion
    const targetBranch = config['target-branch'] || ''
```

Reply:
```suggestion
```